src/components/BattleToolbar.js

```diff
@@ -120,14 +121,14 @@ class BattleToolbar extends Component {
           <div className="battle-toolbar--options-container">
             <button
               title="Options Menu"
-              className={`${buttonClass} battle-toolbar--button__options`}
+              className={`battle-toolbar--button__toggle battle-toolbar--button__options`}
```

Comment:
The `battle-toolbar--button__options` here wasn't actually being used in the CSS so could be removed.